### ISSUE TITLE
UI changes are not reflecting after restart

### ISSUE THREAD

**Issue by GPreksha** [capitalone/Hygieia#1863]
I did background changes to UI in dash.css.. But changes wont reflect in later start of UI.
tried clearing cache and cookies no result 

Any ideas ?

**Comment by matthewbeasley** [comment]
Hey! I have a couple questions:
Are you referring to /UI/src/components/themes/dash.less?
What gulp command are you using to serve the UI?

**Comment by matthewbeasley** [comment]
Awesome. Gulp serve is the correct command. What "gulp serve" does is it takes the files in the UI/src directory, does some processing and then outputs those files to another directory 'dist'. So, if you modify the files in your dist folder they will be overridden the next time gulp runs the "themes" task. For more information see [https://github.com/capitalone/Hygieia/blob/master/UI/gulpfile.js](url) and look for the line that starts `gulp.task('themes', function() {`

tldr; If you want your changes to persist, only modify files in the src directory.
